### ISSUE TITLE
MRXS Slide Ramdisk buffer Bug

### ISSUE THREAD

**Issue by siddhir** [jamesdolezal/slideflow#300]
## Description
During tile extraction the ramdisk buffer feature is not compatible with MRXS format slides given they are made of multiple files unlike .svs

## To Reproduce
1. Extract with ramdisk buffer

## Expected behavior
Flag buffer as incompatible with MRXS files prior to extraction

## Environment:
name: slideflow2
channels:
  - pytorch
  - conda-forge
  - defaults
dependencies:
  - _libgcc_mutex=0.1=main
  - _openmp_mutex=5.1=1_gnu
  - blas=1.0=mkl
  - brotlipy=0.7.0=py39h27cfd23_1003
  - bzip2=1.0.8=h7b6447c_0
  - ca-certificates=2023.01.10=h06a4308_0
  - certifi=2022.9.24=py39h06a4308_0
  - cffi=1.15.1=py39h74dc2b5_0
  - charset-normalizer=2.0.4=pyhd3eb1b0_0
  - cryptography=38.0.1=py39h9ce1e76_0
  - cudatoolkit=11.3.1=h2bc3f7f_2
  - ffmpeg=4.3=hf484d3e_0
  - freetype=2.12.1=h4a9f257_0
  - giflib=5.2.1=h7b6447c_0
  - gmp=6.2.1=h295c915_3
  - gnutls=3.6.15=he1e5248_0
  - intel-openmp=2021.4.0=h06a4308_3561
  - jpeg=9e=h7f8727e_0
  - lame=3.100=h7b6447c_0
  - lcms2=2.12=h3be6417_0
  - ld_impl_linux-64=2.38=h1181459_1
  - lerc=3.0=h295c915_0
  - libblas=3.9.0=12_linux64_mkl
  - libcblas=3.9.0=12_linux64_mkl
  - libdeflate=1.8=h7f8727e_5
  - libffi=3.3=he6710b0_2
  - libgcc-ng=11.2.0=h1234567_1
  - libgomp=11.2.0=h1234567_1
  - libiconv=1.16=h7f8727e_2
  - libidn2=2.3.2=h7f8727e_0
  - liblapack=3.9.0=12_linux64_mkl
  - libpng=1.6.37=hbc83047_0
  - libstdcxx-ng=11.2.0=h1234567_1
  - libtasn1=4.16.0=h27cfd23_0
  - libtiff=4.4.0=hecacb30_1
  - libunistring=0.9.10=h27cfd23_0
  - libuv=1.40.0=h7b6447c_0
  - libwebp=1.2.4=h11a3e52_0
  - libwebp-base=1.2.4=h5eee18b_0
  - lz4-c=1.9.3=h295c915_1
  - mkl=2021.4.0=h06a4308_640
  - mkl-service=2.4.0=py39h7f8727e_0
  - mkl_fft=1.3.1=py39hd3c417c_0
  - mkl_random=1.2.2=py39h51133e4_0
  - ncurses=6.3=h5eee18b_3
  - nettle=3.7.3=hbbd107a_1
  - ninja-base=1.10.2=hd09550d_5
  - numpy-base=1.24.3=py39h31eccc5_0
  - openh264=2.1.1=h4ff587b_0
  - openssl=1.1.1t=h7f8727e_0
  - pip=22.2.2=py39h06a4308_0
  - pycparser=2.21=pyhd3eb1b0_0
  - pyopenssl=22.0.0=pyhd3eb1b0_0
  - pysocks=1.7.1=py39h06a4308_0
  - python=3.9.13=haa1d7c7_2
  - python_abi=3.9=2_cp39
  - pytorch=1.11.0=py3.9_cuda11.3_cudnn8.2.0_0
  - pytorch-mutex=1.0=cuda
  - readline=8.2=h5eee18b_0
  - six=1.16.0=pyhd3eb1b0_1
  - sqlite=3.39.3=h5082296_0
  - tk=8.6.12=h1ccaba5_0
  - torchaudio=0.11.0=py39_cu113
  - torchvision=0.12.0=py39_cu113
  - typing_extensions=4.3.0=py39h06a4308_0
  - tzdata=2022f=h04d1e81_0
  - wheel=0.37.1=pyhd3eb1b0_0
  - xz=5.2.6=h5eee18b_0
  - zlib=1.2.13=h5eee18b_0
  - zstd=1.5.2=ha4553b6_0
  - pip:
    - absl-py==1.3.0
    - affine==2.3.1
    - alabaster==0.7.12
    - alembic==1.10.3
    - anyio==3.6.2
    - argon2-cffi==21.3.0
    - argon2-cffi-bindings==21.2.0
    - array-record==0.2.0
    - arrow==1.2.3
    - asciitree==0.3.3
    - astor==0.8.1
    - asttokens==2.2.1
    - astunparse==1.6.3
    - attrs==22.1.0
    - autograd==1.5
    - autograd-gamma==0.5.0
    - babel==2.11.0
    - backcall==0.2.0
    - beautifulsoup4==4.12.0
    - biscuit==0.0.1
    - bleach==6.0.0
    - blis==0.7.9
    - cachetools==5.2.0
    - catalogue==2.0.8
    - cellpose==2.1.1
    - chardet==3.0.4
    - click==8.1.3
    - click-plugins==1.1.1
    - cligj==0.7.2
    - cloudpickle==2.2.0
    - cmaes==0.9.1
    - colorlog==6.7.0
    - comm==0.1.3
    - commonmark==0.9.1
    - confection==0.0.4
    - configspace==0.6.0
    - contourpy==1.0.6
    - cplex==22.1.1.0
    - cramjam==2.6.2
    - crc32c==2.3
    - cucim==23.2.0
    - cupy-cuda11x==11.6.0
    - cvxpy==1.3.1
    - cycler==0.11.0
    - cymem==2.0.7
    - cython==0.29.32
    - dask==2022.10.2
    - debugpy==1.6.6
    - decorator==5.1.1
    - defusedxml==0.7.1
    - distributed==2022.10.2
    - dm-tree==0.1.7
    - docutils==0.19
    - ecos==2.0.12
    - emcee==3.1.3
    - entrypoints==0.4
    - etils==1.2.0
    - executing==1.2.0
    - fastai==2.7.11
    - fastcore==1.5.28
    - fastdownload==0.0.7
    - fasteners==0.18
    - fastjsonschema==2.16.3
    - fastparquet==2023.2.0
    - fastprogress==1.0.3
    - fastremap==1.13.4
    - fastrlock==0.8.1
    - filelock==3.12.0
    - flatbuffers==1.12
    - fonttools==4.38.0
    - formulaic==0.5.2
    - fpdf==1.7.2
    - fpdf2==2.7.3
    - fqdn==1.5.1
    - fsspec==2022.10.0
    - future==0.18.2
    - gast==0.4.0
    - gitdb==4.0.10
    - gitpython==3.1.31
    - glfw==2.5.5
    - google-auth==2.14.1
    - google-auth-oauthlib==0.4.6
    - google-pasta==0.2.0
    - googleapis-common-protos==1.59.0
    - gptrepo==1.0.2
    - greenlet==2.0.2
    - grpcio==1.50.0
    - h5py==3.7.0
    - heapdict==1.0.1
    - huggingface-hub==0.14.1
    - idna==2.8
    - imagecodecs==2023.3.16
    - imageio==2.22.4
    - imageio-ffmpeg==0.4.7
    - imagesize==1.4.1
    - imgui==1.4.1
    - importlib-metadata==5.0.0
    - importlib-resources==5.12.0
    - interface-meta==1.3.0
    - ipykernel==6.22.0
    - ipython==8.11.0
    - ipython-genutils==0.2.0
    - ipywidgets==8.0.5
    - isoduration==20.11.0
    - jedi==0.18.2
    - jinja2==3.1.2
    - joblib==1.2.0
    - jsonpointer==2.3
    - jsonschema==4.17.3
    - jupyter==1.0.0
    - jupyter-client==8.1.0
    - jupyter-console==6.6.3
    - jupyter-core==5.3.0
    - jupyter-events==0.6.3
    - jupyter-server==2.5.0
    - jupyter-server-terminals==0.4.4
    - jupyterlab-pygments==0.2.2
    - jupyterlab-widgets==3.0.6
    - keras==2.9.0
    - keras-preprocessing==1.1.2
    - kiwisolver==1.4.4
    - langcodes==3.3.0
    - libclang==14.0.6
    - lifelines==0.27.3
    - llvmlite==0.39.1
    - locket==1.0.0
    - mako==1.2.4
    - markdown==3.4.1
    - markupsafe==2.1.1
    - matplotlib==3.6.2
    - matplotlib-inline==0.1.6
    - mistune==2.0.5
    - msgpack==1.0.4
    - munch==2.5.0
    - murmurhash==1.0.9
    - natsort==8.3.1
    - nbclassic==0.5.3
    - nbclient==0.7.2
    - nbconvert==7.2.10
    - nbformat==5.8.0
    - nest-asyncio==1.5.6
    - networkx==2.8.8
    - ninja==1.11.1
    - notebook==6.5.3
    - notebook-shim==0.2.2
    - numba==0.56.4
    - numcodecs==0.11.0
    - numpy==1.21.6
    - oauthlib==3.2.2
    - opencv-python==4.6.0.66
    - opencv-python-headless==4.7.0.72
    - opt-einsum==3.3.0
    - optuna==3.1.1
    - osqp==0.6.2.post9
    - packaging==21.3
    - pandas==1.5.1
    - pandocfilters==1.5.0
    - parameterized==0.8.1
    - parse==1.12.1
    - parso==0.8.3
    - partd==1.3.0
    - pathy==0.10.1
    - pbr==5.11.0
    - pexpect==4.8.0
    - pickleshare==0.7.5
    - pillow==9.5.0
    - pkgconfig==1.5.5
    - platformdirs==3.2.0
    - ply==3.11
    - preshed==3.0.8
    - pretrainedmodels==0.7.4
    - prometheus-client==0.16.0
    - promise==2.3
    - prompt-toolkit==3.0.38
    - protobuf==3.20.2
    - psutil==5.9.4
    - ptyprocess==0.7.0
    - pure-eval==0.2.2
    - pyarrow==10.0.0
    - pyasn1==0.4.8
    - pyasn1-modules==0.2.8
    - pydantic==1.10.7
    - pygments==2.13.0
    - pynisher==0.6.4
    - pynndescent==0.5.8
    - pyomo==6.5.0
    - pyopengl==3.1.6
    - pyparsing==3.0.9
    - pyperclip==1.8.2
    - pyrfr==0.8.3
    - pyrsistent==0.19.3
    - python-dateutil==2.8.2
    - python-json-logger==2.0.7
    - pytz==2022.6
    - pyvips==2.2.1
    - pywavelets==1.4.1
    - pyyaml==6.0
    - pyzmq==25.0.2
    - qdldl==0.1.7
    - qtconsole==5.4.1
    - qtpy==2.3.0
    - rasterio==1.3.3
    - regex==2022.10.31
    - requests==2.22.0
    - requests-oauthlib==1.3.1
    - requests-wsgi-adapter==0.4.1
    - rfc3339-validator==0.1.4
    - rfc3986-validator==0.1.1
    - rich==12.6.0
    - rsa==4.9
    - saliency==0.2.0
    - scikit-image==0.19.3
    - scikit-learn==1.1.3
    - scikit-misc==0.1.4
    - scipy==1.9.3
    - scs==3.2.3
    - seaborn==0.11.2
    - send2trash==1.8.0
    - setuptools==67.7.2
    - shapely==1.8.5.post1
    - slideflow==2.0.3.post1
    - smac==1.4.0
    - smart-open==6.3.0
    - smmap==5.0.0
    - sniffio==1.3.0
    - snowballstemmer==2.2.0
    - snuggs==1.4.7
    - sortedcontainers==2.4.0
    - soupsieve==2.4
    - spacy==3.5.1
    - spacy-legacy==3.0.12
    - spacy-loggers==1.0.4
    - sphinx==5.3.0
    - sphinx-markdown-tables==0.0.17
    - sphinxcontrib-applehelp==1.0.2
    - sphinxcontrib-devhelp==1.0.2
    - sphinxcontrib-htmlhelp==2.0.0
    - sphinxcontrib-jsmath==1.0.1
    - sphinxcontrib-qthelp==1.0.3
    - sphinxcontrib-serializinghtml==1.1.5
    - sphinxcontrib-video==0.0.1.dev3
    - sqlalchemy==2.0.9
    - srsly==2.4.6
    - stack-data==0.6.2
    - tabulate==0.9.0
    - tblib==1.7.0
    - tensorboard==2.9.1
    - tensorboard-data-server==0.6.1
    - tensorboard-plugin-wit==1.8.1
    - tensorflow==2.9.2
    - tensorflow-datasets==4.9.2
    - tensorflow-estimator==2.9.0
    - tensorflow-io-gcs-filesystem==0.27.0
    - tensorflow-metadata==1.13.1
    - tensorflow-probability==0.17.0
    - termcolor==2.1.0
    - terminado==0.17.1
    - thinc==8.1.9
    - threadpoolctl==3.1.0
    - tifffile==2022.10.10
    - tinycss2==1.2.1
    - toml==0.10.2
    - toolz==0.12.0
    - tornado==6.1
    - tqdm==4.64.1
    - traitlets==5.9.0
    - typer==0.7.0
    - umap-learn==0.5.3
    - uri-template==1.2.0
    - urllib3==1.25.11
    - versioneer==0.28
    - wasabi==1.1.1
    - wcwidth==0.2.6
    - webcolors==1.13
    - webencodings==0.5.1
    - webob==1.8.5
    - websocket-client==1.5.1
    - werkzeug==2.2.2
    - widgetsnbextension==4.0.6
    - wrapt==1.14.1
    - zarr==2.14.2
    - zict==2.2.0
    - zipp==3.10.0

**Comment by jamesdolezal** [comment]
Thanks, this has been fixed on `dev-2.1`, and will be incorporated into the 2.1 version release.
